racket/collects/racket/vector.rkt

```diff
@@ -94,7 +94,9 @@
 ;; uses name for error reporting
 (define (one-arg-check f v name)
   (unless (and (procedure? f) (procedure-arity-includes? f 1))
-    (raise-argument-error name "(any/c . -> . any/c)" 0 f)))
+    (raise-argument-error name "(any/c . -> . any/c)" 0 f))
+  (unless (vector? v)
+    (raise-argument-error name "vector?" v)))
```

Comment:
For consistency with the above `raise-argument-error`, let's also use the second form. That is, include `1` as `bad-pos`, and also add `f` as an argument before the last argument.